spec/depicts/transport/aircraft.yaml

```diff
@@ -19,7 +19,7 @@
   category: Category:Seaplanes
   excludeRegex: "/(Videos)/i"
 
-- depictsId: Q8492796
+- depictsId: Q180173
```

Comment:
Might it be better to use the slightly higher class?
Q2165278 = glider.
Q180173 = type of glider aircraft used in the sport of gliding

Reply:
I was trying to understand difference between both of them, but for me description was "its the same", and in Polish there is probably no difference. Thank you for fixing this.

Reply:
The best thing to look at is most likley the subclass of and instance of relationships.
Perhaps I should write a little doc on adding new depicts and what to look out for :)